packages/html-reporter/src/testFileView.tsx

```diff
@@ -59,7 +59,7 @@ export const TestFileView: React.FC<React.PropsWithChildren<{
           <span data-testid='test-duration' style={{ minWidth: '50px', textAlign: 'right' }}>{msToString(test.duration)}</span>
         </div>
         <div className='test-file-details-row'>
-          <Link href={`#?testId=${test.testId}`} title={[...test.path, test.title].join(' › ')} className='test-file-path-link'>
+          <Link href={testResultHref({ test })} title={[...test.path, test.title].join(' › ')} className='test-file-path-link'>
```

Comment:
I wonder whether `testResultHref()` should automatically append `filterParam`? For example, here and below at line 78 we lose it, which looks unintentional.

Reply:
Sounds good. I'd rather fix that in a separate PR though, so it's easier to revert if we don't want it for some reason.